payments-core/src/main/java/com/stripe/android/auth/PaymentBrowserAuthContract.kt

```diff
@@ -26,7 +26,7 @@ internal class PaymentBrowserAuthContract :
     ): Intent {
         val defaultReturnUrl = DefaultReturnUrl.create(context)
         val shouldUseBrowser =
-            input.hasDefaultReturnUrl(defaultReturnUrl) || input.isInstantApp
+            input.hasDefaultReturnUrl(defaultReturnUrl) || input.isInstantApp || input.returnUrl == null
```

Comment:
This is required so that the StripeBrowserActivity is used instead of the webview.

If the webview is used, and it is canceled, then the outcome is Canceled and in StripeIntentResult when you get the outcome it isn't = to Unknown, so it doesn't look at LUXE to determine the next action support where it would find this is a success case and the paymentsheet should close.